### DISCUSSION TITLE
Remember me not working on organization identity-first login

### DISCUSSION THREAD

**Discussion by nicymike** [keycloak/keycloak#35955] [Q&A]
Hi, I switched to the new organization feature but customers complained that remember me is no longer working. I digged into the issue and this is what I found.

Switching to organizations means switching to identity-first login which separates email and password. The checkbox for remember me is on the first screen where the email must be entered. I am using the default keycloak v2 theme.

![image](https://github.com/user-attachments/assets/9b2c507f-bb0c-4387-8f85-399de7533fcc)

![image](https://github.com/user-attachments/assets/78121a39-f332-414c-852c-ef42d0f7026d)

Activating the checkbox on the first screen has no effect because the flag will be ignored on server side.

![image](https://github.com/user-attachments/assets/30cf1d61-1e6e-4dd6-bcc4-358c03bacec1)

![image](https://github.com/user-attachments/assets/6820cff4-89bf-4fee-a908-cbd0b319eab7)

There is no checkbox on the second screen but if i alter the theme and add the checkbox it is working again. 

What is your opinion on this? Either submitting the first screen should persist the value of the checkbox somehow or the checkbox should be moved to the second screen. Remember I am using the default keycloak v2 theme.

Thank you for your help.
Michael

**Comment by sebnuss** [comment]
We have seen the same problem. Since we have organizations enabled, we have desabled "Remember me" for the time being.

**Comment by cheindel** [comment]
Same problem here. I would consider this a bug. It should work the same as without activated organization feature and checkbox should stay on first page as is.

**Comment by ThomasRFW** [comment]
I think this is a bug. if remember me means store the e-mail  address or login name.

**Comment by clasbleiz** [comment]
Hi,
We’ve also noticed that the "Remember Me" feature is not working since we switched to the new organization feature.
Is this a known limitation that is still ongoing?
Is there a way to work around this issue?
Thanks a lot

**Comment by Esurnir** [comment]
An issue for this has been created : #47017

**Comment by Esurnir** [comment]
This issue has now been resolved with the release 26.7.0
